### PR TITLE
Fixes favicon not loaded when using a subdirectory

### PR DESCRIPTION
**Pull request by HorlogeSkynet**
Hey there :wave:

When Apaxy is used within a subdirectory (let's say `/share`), browsers would still try to load the favicon from `/`.
This patch moves the favicon file beside its friend "theme" files and deals with the `IndexHeadInsert` Apache directive to point to the correct location of the set favicon.

Bye :bowing_man:

### CODE REVIEW

File: apaxy/htaccess.txt [hunk 1]
```diff
@@ -19,11 +19,13 @@ Options +Indexes
 # index options
 IndexOptions +Charset=UTF-8 +FancyIndexing +FoldersFirst +HTMLTable +IconsAreLinks +IgnoreCase +NameWidth=* +SuppressDescription +SuppressRules +XHTML
 
-# meta viewport
-IndexHeadInsert "<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\" />"
+# favicon & meta viewport
+IndexHeadInsert "\
+        <link rel=\"shortcut icon\" href=\"{FOLDERNAME}/theme/favicon.ico\">\
```
Comment (oupala) [inline apaxy/htaccess.txt]: Could ou please close the link tag with a trailing ` />` as in the line 25?

**Comment by oupala** [comment]
This is a great commit, I tested it and it works perfectly. I am willing to merge it.

I made a suggestion to improve your code (see in the commit tab).

And can you also rewrite your commit message to follow conventional changelog, as stated in the [contributing doc](https://github.com/oupala/apaxy/blob/develop/CONTRIBUTING.md#closing-merge-requests).

**Comment by HorlogeSkynet** [comment]
Here you go !
My text editor caught a trailing space, I added it to my branch too.
Bye :wave:

**Comment by oupala** [comment]
Ok, merged!

By the way, it could be a good occasion to redesign the favicon, which I made from the original logo of apaxy (see [apaxy homepage](https://oupala.github.io/apaxy/)):

apaxy logo

The logo is nice but the problem is that it is not very visible as a favicon. Does anyone has a suggestion?

**Comment by HorlogeSkynet** [comment]
Thanks @oupala !

I agree with you about its visibility. I'll try to ping a friend about it.
Bye :wave:

**Pull request merged**